setup-tdx-host.sh

```diff
@@ -22,9 +35,22 @@ EOF
 
 apt update
 
-# --allow-downgrades : if kobuk-tdx-host is already installed
-apt install --yes --allow-downgrades kobuk-tdx-host || _error "Cannot install TDX components !"
-apt upgrade --yes --allow-downgrades kobuk-tdx-host || _error "Cannot upgrade TDX components !"
+apt install --yes --allow-downgrades \
+    linux-intel-opt \
+    qemu-system-x86 \
+    libvirt-daemon-system \
+    libvirt-clients \
+    ovmf \
+    tdx-tools-host
+
+# detect the intel-opt current release (example : 6.5.0-1003-intel-opt)
+KERNEL_RELEASE=$(apt show linux-image-intel-opt 2>&1 | gawk 'match($0, /Depends: linux-image-(.+)/, a) {print a[1]}')
```

Comment:
It looks like `gawk` comes installed on Ubuntu Server and I think it's safe to assume that won't change in future releases, but if you wanted to be extra safe we could `apt install` it above.